lib/page/forget.dart

```diff
@@ -0,0 +1,100 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import '../auth/forget/bloc/forget_state.dart';
+import '../auth/forget/data/forget_repository.dart';
+import '../common/components/forms/custom_form_input_class.dart';
+import '../common/config/enum.dart';
+import '../auth/forget/bloc/forget_cubit.dart';
+import '../auth/forget/data/forget_model.dart';
+import '../common/components/forms/form_component.dart';
+import 'register.dart';
+
+class Forget extends StatefulWidget {
+  const Forget({Key? key}) : super(key: key);
+  static const routeName = '/forget';
+
+  @override
+  State<Forget> createState() => _ForgetState();
+}
+
+class _ForgetState extends State<Forget> {
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider(
+      create: (context) => ForgetCubit(ForgetRepositoryImpl()),
+      child: Scaffold(
+        resizeToAvoidBottomInset: true,
+        body: Container(
+          color: Theme.of(context).colorScheme.primary,
+          child: SafeArea(child: buildForget()),
+        ),
+      ),
+    );
+  }
+
+  Widget buildForget() {
+    return ListView(
+      padding: EdgeInsets.zero,
+      children: [
+        Padding(
+          padding: const EdgeInsets.only(top: 29.0, bottom: 45.0),
+          child: Center(
+              child:
+                  Image.asset('lib/common/assets/images/GlobeIconSmall.png')),
+        ),
+        BlocBuilder<ForgetCubit, ForgetState>(builder: (context, state) {
+          if (state is ForgetSuccessState) {
+            return const Text("success");
```

Comment:
I think you need to change the success handler, as of now it will just show success text, might want to redirect the page.

Reply:
alright i've changed it to redirect to another page (though its still a dummy because there's no login page yet)

Reply:
nice